packages/libs/src/sdk/api/albums/AlbumsApi.ts

```diff
@@ -243,6 +243,105 @@ export class AlbumsApi {
     )
   }
 
+  /**
+   * Gets the Solana instructions for a purchase as a gift
+   *
+   * @hidden
+   */
+  async getGiftAlbumInstructions(params: GetPurchaseAlbumTransactionRequest) {
```

Comment:
Why is this necessary? The existing `getPurchaseTrackTransaction` and it's album equivalent do the same thing, no? You can extract the instructions from there and exclude the transfer. Alternatively, if you want something like this, we can make the other methods pull the instructions from here. This feels too wet

Reply:
they are slightly different. i disagree kind of... prefer the wet to the 3 layers of nesting where each layer is exposed